.github/fixtures/test-keep-a-changelog-links-latest-arg/cliff.toml

```diff
@@ -0,0 +1,79 @@
+# configuration file for git-cliff (0.1.0)
+
```

Comment:
We can probably remove the header from the keep-a-changelog related fixtures - we don't use it for the other config files as well.

Reply:
I will do that 👍

Reply:
Great!

Reply:
Done :)